woocommerce/woocommerce-android
# [Shipping Labels Revamp] Add Improvements and bug fixes to Packages UI

Summary
==========
Add some adjustments to the Package UI selection:
- Introduce a working Selector for the Package Type
- Add index resilience to the Saved and Carrier package list to avoid race conditions causing Index errors
- Introduce Fragment Result handling to allow the main Shipping Labels form to receive the selected Package Data
- Unify the ViewModel Package selection event system to a simplified one for better control over the Package data flow

How to Test
==========
1. As the Package selection system is not yet integrated with real endpoints, it's not possible to effectively test the features introduced here. The unit tests are the only way to verify the behaviors here, which is why this work is still behind a feature flag.

Update release notes:


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
